.tekton/osc-caa-pull-request.yaml

```diff
@@ -29,6 +29,8 @@ spec:
     value: cloud-api-adaptor/Dockerfile.openshift
   - name: path-context
     value: src
+  - name: prefetch-input
```

Comment:
Suggestion for the commit message (from your commits in `sandboxed-container-operator`):

```
Following Konflux doc:
- https://konflux-ci.dev/docs/building/prefetching-dependencies/#enabling-prefetch-builds-for-rpm
- https://konflux-ci.dev/docs/building/activation-keys-subscription/#configuring-an-rpm-lockfile-for-hermetic-builds
```

Reply:
Done